docs/connectors/taps/google_analytics.rst

```diff
@@ -0,0 +1,79 @@
+
+.. _tap-google-analytics:
+
+Tap Google Analytic
+-----------
+
+
+Configuring what to replicate
+'''''''''''''''''''''''''''''
+
+PipelineWise configures every tap with a common structured YAML file format.
+A sample YAML for Google Analytic replication can be generated into a project directory by
+following the steps in the :ref:`generating_pipelines` section.
+
```

Comment:
Can you please add a few sentences about authentication methods (OAuth and service accouns) to docs/connectors/taps/google_analytics.rst. This is well detailed in the tap README.md but would be nice to have something in the PipelineWise doc as well

Also, can you please mention that STATE messages and incremental replication is currently not supported by this tap? This is quite important and documented int the tap README.md but would be nice to clarify this in the pipelinewise documentation at well.